packages/commonwealth/client/index.html

```diff
@@ -3,27 +3,21 @@
   <head>
     <meta charset="utf-8" />
     <title>Common</title>
-    <base href="/" />
```

Comment:
This line was added in the previous PR but is no longer needed. This is because the vite config is now in the correct directory.